noxfile.py

```diff
@@ -15,17 +15,17 @@
 
 
 @nox.session(py=ALL_INTERPRETERS)
-def unit(session):
+def tests(session):
     session.install("-e", ".[testing]")
     session.run(
         "pytest",
         f"--cov={CODE}",
-        "--cov=tests.unit",
+        "--cov=gridded_etl_tools",
         "--cov-append",
         "--cov-config",
         HERE / ".coveragerc",
         "--cov-report=term-missing",
-        "tests/unit",
+        "tests",
```

Comment:
The unit tests are distinct from the system tests. We don't use them to measure coverage and the system tests are expected to take longer to run and require extra setup beyond what can be scripted in Nox, so we make them optional.